library/spark/spark-accumulo-library/pom.xml

```diff
@@ -65,6 +67,43 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>net.alchim31.maven</groupId>
```

Comment:
What is this plugin used for? It will need to be added to the NOTICES file if we need it.

Reply:
Done.